drivers/display/Kconfig.gc9a01a

```diff
@@ -0,0 +1,12 @@
+# GC9A01A display driver configuration options
+
+# SPDX-License-Identifier: Apache-2.0
+
+config GC9A01A
+	bool "GC9A01A display driver"
+	default y
+	depends on DT_HAS_WAVESHARE_GC9A01A_ENABLED
```

Comment:
We do not implement drivers for displays or display resellers, but for display controller. Here, it cannot be `DT_HAS_WAVESHARE_GC9A01A_ENABLED`, maybe `DT_HAS_GALAXYCORE_GC9A01A_ENABLED`